mldsa/sign.c

```diff
@@ -2,6 +2,7 @@
  * Copyright (c) The mldsa-native project authors
  * SPDX-License-Identifier: Apache-2.0 OR ISC OR MIT
  */
+#include <stdbool.h>
```

Comment:
Can you please eliminate stdbool, please?
We don't use `stdbool.h` anywhere in mlkem-native/mldsa-native. It's not part of C90 and I don't see a good reason to use it here.

Reply:
It's needed to get the literal "true". I guess I could just replace that by "1".

Reply:
Didn't realize it wasn't in C90...

Reply:
Yes, please replace by 1.
stdbool got added in C99. Same applies to stdint - but that is basically impossible to eliminate.